scania-digital-design-system/tegel
# fix(table): renaming data-table to table

**Describe pull-request**  
Renaming Data table to Table

**How to test**  
1. Run the Storybook link
2. There should be no mention of Data table any more
3. Check the branch on your local
4. There should be no variables, comments or any other trail of "data table"

**Suggested test steps**

**Additional context**  
Prefix change from sdds to tds will be done in a separate PR due to too many files affected and wish to keep a list of changes bearable.
